### DISCUSSION TITLE
Auto-session will cause error when staring DAP server .

### DISCUSSION THREAD

**Discussion by BilderLoong** [jbyuki/one-small-step-for-vimkind#38] [General]
Using the [auto-session](https://github.com/rmagatti/auto-session) plugin seems to cause this [Neovim is waiting for input at startup. Aborting.
](https://github.com/jbyuki/one-small-step-for-vimkind#neovim-is-waiting-for-input-at-startup-aborting) when run  `:lua require"osv".launch({port=8086})`.

I tried `nvim --headless` there is no error.

**Comment by jbyuki** [comment]
Interesting, can you provide me with a few more informations.

1. How do you launch nvim ? (Ex. through the terminal, with a launch script, ...)
2.  What is your configuration for auto-session.
3. Neovim version

**Comment by jbyuki** [comment]
I tried installing `auto-session` and started a debugging session without issue.
More informations would be helpful.

Note that I've noticed `auto-session` is path-sensitive, meaning sensitive in which folder nvim is opened for its autosession mechanism. `one-small-step-for-vimkind` will internally open an nvim instance when invoking `launch()` and it is opened in the cwd.  So the error might only appear in that directory.
